.eslintrc

```diff
@@ -0,0 +1,26 @@
+{
+  "extends": "airbnb/base",
+  "env": {
+    "mocha": true,
+    "browser": true,
+    "node": true,
+    "es6": true
+  },
+  "rules": {
+    "comma-dangle": [0, "always-multiline"],
+    "no-use-before-define": [2, "nofunc"],
+    "no-underscore-dangle": ["off", {"allow": ["_id"]}],
+    "spaced-comment": [0, "never"],
+    "array-callback-return": [0],
+    "arrow-body-style": [0, "as-needed"],
+    "new-cap": [0, {"newIsCap": false}],
+    "no-param-reassign": 0,
+    "max-len": ["error", 200]
+  },
+  "globals": {
+    "sinon": false,
+    "expect": false,
+    "angular": false,
```

Comment:
good catch. I'll remove all of the globals because I changed the way that I'm pulling in the test libraries. These were a carry over from when I was using webpack.